querygen/userjourney_fs.go

```diff
@@ -27,6 +27,10 @@ var UserJourneyFieldSet = FieldSet{
 		subObj:    "properties",
 		srcFields: []string{"responseObject.metadata.labels.appstudio.openshift.io/component"},
 	},
+	"vulnerabilities_critical": {subObj: "properties", srcFields: []string{"vulnerabilities.critical"}},
+	"vulnerabilities_high":     {subObj: "properties", srcFields: []string{"vulnerabilities.high"}},
+	"vulnerabilities_medium":   {subObj: "properties", srcFields: []string{"vulnerabilities.medium"}},
+	"vulnerabilities_low":      {subObj: "properties", srcFields: []string{"vulnerabilities.low"}},
```

Comment:
These `srcFields` also need updating to match the new field names. E.g. `clair_scan_result.vulnerabilities.critical`.

Be sure to run the `fetch-uj-records.sh` script to verify that all the expected fields are populated for this query.